src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -132,9 +142,7 @@ def _compute_grants(
         return grants
 
     def _migrate_table(
-        self,
-        src_table: TableToMigrate,
-        grants: list[Grant] | None = None,
+        self, src_table: TableToMigrate, grants: list[Grant] | None = None, mounts: Iterable[Mount] | None = None
```

Comment:
```suggestion
        self, src_table: TableToMigrate, grants: list[Grant], mounts: list[Mount]
```
pass empty lists that are always there. don't do `| None` for collection types, pass in empty collection instead.

Reply:
fixed